src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -112,7 +125,7 @@ def __init__(
 
         default_kwargs = {
             "time_limit": 60,
-            "excluded_model_types": ["KNN", "XT", "RF"],
+            # "excluded_model_types": ["KNN", "XT", "RF"],
```

Comment:
Is this intended?

Reply:
yeah this is intended. If we exclude "RF", then no quantile model could be used. I'm curious why we need to exclude some models at initial. That also why I didn't remove this line of code

Reply:
I believe these models are removed because usually they do not give a better results than DL models while training them is going to consume a lot of space. Do you mean that if we do not enable RF then we will not be able to use the quantiles?

Reply:
yes. If we don't enable RF then there's no quantile model available.